doc/authentication.md

```diff
@@ -0,0 +1,91 @@
+
+Cockpit Authentication
+================================
+
+Cockpit authorizes users by looking at the Authorization header on requests
+to /login. Cockpit will atempt to perform the authentication action that
+is configured for the auth scheme contained in the header.
+
+Cockpit supports the following authentication actions:
+
+ * spawn-authenticator-with-header
```

Comment:
I realize i mistyped. But shouldn't these two be consistent? Maybe both should start with 'spawn-login-'?